html.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = `
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charSet="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>SVG to GeoJSON</title>
+    <script src='https://hey.mapbox.com/build/hey.js'></script>
```

Comment:
What's this?

Reply:
I carried this over from a couple other examples I've seen, but I don't think it actually exists and I'm going to remove it.